.gitignore

```diff
@@ -1 +1,94 @@
 /target/
+
+
+# Created by https://www.toptal.com/developers/gitignore/api/maven,eclipse
```

Comment:
Added more to gitignore so that any metadata that is not actually needed for the project and stuff like that is ignored when changes to those are made